ludwig/utils/batch_size_tuner.py

```diff
@@ -51,7 +52,9 @@ def _is_valid_batch_size(batch_size):
             gc.collect()
 
             try:
-                samples_per_sec = self.evaluate(batch_size, total_steps=5)
+                samples_per_sec = self.evaluate(
+                    batch_size, total_steps=5, global_max_sequence_length=global_max_sequence_length
```

Comment:
nit: Make this a constant in the file.

Reply:
I'm not quite sure I understand. Why should `samples_per_sec` be a constant?

Reply:
Ohh I see